tests/core/pyspec/eth2spec/test/fulu/unittests/das/test_das.py

```diff
@@ -149,3 +157,315 @@ def test_get_extended_sample_count__table_in_spec(spec):
             spec.get_extended_sample_count(allowed_failures=allowed_failures)
             == expected_extended_sample_count
         )
+
+
+def run_is_data_available_peerdas_test(spec, blob_data):
+    def callback():
+        yield spec.is_data_available(spec.Root(b"\x00" * 32))
+
+    return next(with_blob_data(spec, blob_data, callback))
+
+
+@with_fulu_and_later
+@spec_state_test
+def test_is_data_available_peerdas__valid(spec, state):
+    rng = random.Random(1234)
+    _, blobs, blob_kzg_proofs, _, sidecars = get_block_with_blob_and_sidecars(
+        spec, state, rng=rng, blob_count=2
+    )
+    blob_data = BlobData(blobs, blob_kzg_proofs, sidecars)
+
+    result = run_is_data_available_peerdas_test(spec, blob_data)
+
+    assert result is True, "Data should be available for the block with blob data"
+
+
+@with_fulu_and_later
+@spec_state_test
+def test_is_data_available_peerdas__not_avail(spec, state):
```

Comment:
Just a nit, let's use the full word.
```suggestion
def test_is_data_available_peerdas__not_available(spec, state):
```